en/docs/setup/token-persistence.md

```diff
@@ -105,10 +66,6 @@ caused by the client application mishandling the
 
 -   [CONN\_APP\_KEY
     constraint](#TokenPersistence-CONN_APP_KEYconstraint)
```

Comment:
these links get reverted in this PR and will be broken again.

Reply:
Fixed: 29b93d6c04c67139b6ba3aa772d38d24cd51e164